e2e/cypress/support/api_commands.js

```diff
@@ -1,374 +0,0 @@
-        return cy.wrap(response);
-    });
-});
-
-Cypress.Commands.add('apiLinkGroup', (groupID) => {
-    return linkUnlinkGroup(groupID, 'POST');
-});
-
-Cypress.Commands.add('apiUnlinkGroup', (groupID) => {
-    return linkUnlinkGroup(groupID, 'DELETE');
-});
-
-function linkUnlinkGroup(groupID, httpMethod) {
-    return cy.request({
-        headers: {'X-Requested-With': 'XMLHttpRequest'},
-        url: `/api/v4/ldap/groups/${groupID}/link`,
-        method: httpMethod,
-        timeout: 60000,
-    }).then((response) => {
-        expect(response.status).to.be.oneOf([200, 201, 204]);
-        return cy.wrap(response);
-    });
-}
-
-Cypress.Commands.add('apiGetGroupTeams', (groupID) => {
-    return getGroupSyncables(groupID, 'team');
-});
-
-Cypress.Commands.add('apiGetGroupTeam', (groupID, teamID) => {
-    return getGroupSyncable(groupID, 'team', teamID);
-});
-
-Cypress.Commands.add('apiGetGroupChannels', (groupID) => {
-    return getGroupSyncables(groupID, 'channel');
-});
-
-Cypress.Commands.add('apiGetGroupChannel', (groupID, channelID) => {
-    return getGroupSyncable(groupID, 'channel', channelID);
-});
-
-function getGroupSyncable(groupID, syncableType, syncableID) {
-    return cy.request({
-        headers: {'X-Requested-With': 'XMLHttpRequest'},
-        url: `/api/v4/groups/${groupID}/${syncableType}s/${syncableID}`,
-        method: 'GET',
-        timeout: 60000,
-    }).then((response) => {
-        expect(response.status).to.equal(200);
-        return cy.wrap(response);
-    });
-}
-
-function getGroupSyncables(groupID, syncableType) {
-    return cy.request({
-        headers: {'X-Requested-With': 'XMLHttpRequest'},
-        url: `/api/v4/groups/${groupID}/${syncableType}s?page=0&per_page=100`,
-        method: 'GET',
-        timeout: 60000,
-    }).then((response) => {
-        expect(response.status).to.equal(200);
-        return cy.wrap(response);
-    });
-}
-
-Cypress.Commands.add('apiUnlinkGroupTeam', (groupID, teamID) => {
-    return linkUnlinkGroupSyncable(groupID, teamID, 'team', 'DELETE');
-});
-
-Cypress.Commands.add('apiLinkGroupTeam', (groupID, teamID) => {
-    return linkUnlinkGroupSyncable(groupID, teamID, 'team', 'POST');
-});
-
-Cypress.Commands.add('apiUnlinkGroupChannel', (groupID, channelID) => {
-    return linkUnlinkGroupSyncable(groupID, channelID, 'channel', 'DELETE');
-});
-
-Cypress.Commands.add('apiLinkGroupChannel', (groupID, channelID) => {
-    return linkUnlinkGroupSyncable(groupID, channelID, 'channel', 'POST');
-});
-
-function linkUnlinkGroupSyncable(groupID, syncableID, syncableType, httpMethod) {
-    return cy.request({
-        headers: {'X-Requested-With': 'XMLHttpRequest'},
-        url: `/api/v4/groups/${groupID}/${syncableType}s/${syncableID}/link`,
-        method: httpMethod,
-        timeout: 60000,
-        body: {auto_add: true},
-    }).then((response) => {
-        expect(response.status).to.be.oneOf([200, 201, 204]);
-        return cy.wrap(response);
-    });
-}
```

Comment:
Maybe deleting this file is not intentional?

Reply:
No - mistake on my part.